core/src/main/scala/org/apache/spark/storage/StorageLevel.scala

```diff
@@ -172,6 +173,7 @@ object StorageLevel {
     case "NONE" => NONE
     case "DISK_ONLY" => DISK_ONLY
     case "DISK_ONLY_2" => DISK_ONLY_2
+    case "DISK_ONLY_3" => DISK_ONLY_3
```

Comment:
Maybe we should fix Java, Python and R sides and the documentation together given that the changes are not so big. I checked it in https://github.com/apache/spark/search?q=DISK_ONLY_2&unscoped_q=DISK_ONLY_2.
BTW, is it worth adding this? It was already possible before via using user-defined `StorageLevel`, and adding this seems making this list inconsistent.